src/main/java/lottoGame/controller/LottoStore.java

```diff
@@ -1,60 +1,74 @@
 package lottoGame.controller;
 
 import static lottoGame.view.Casher.askBeforeWinNums;
+import static lottoGame.view.Casher.askBonusNum;
 import static lottoGame.view.Casher.askBuyPrice;
 import static lottoGame.view.Casher.informBuyCount;
 import static lottoGame.view.Casher.informPublishedLottos;
-import static lottoGame.view.Casher.informWinningLottoNums;
+import static lottoGame.view.Casher.informWinResult;
 
+import java.util.List;
+import lottoGame.model.lotto.Lotto;
 import lottoGame.model.lotto.LottoMachine;
+import lottoGame.model.lotto.LottoNum;
 import lottoGame.model.lotto.Lottos;
 import lottoGame.model.price.BuyPrice;
-import lottoGame.model.winner.BeforeWinNums;
 import lottoGame.model.winner.WinnerResult;
-import lottoGame.view.WinResultDto;
 
 public class LottoStore {
-    public static final int PER_LOTTO_PRICE = 1000;
+
+    public static final int PER_LOTTO_PRICE = 1_000;
 
     public void start() {
         BuyPrice buyPrice = getBuyPrice();
+        LottoMachine lottoMachine = new LottoMachine(PER_LOTTO_PRICE);
 
-        Lottos lottos = buyLottos(buyPrice);
+        Lottos lottos = buyLottos(
+                lottoMachine,
+                buyPrice
+        );
```

Comment:
```suggestion
        Lottos lottos = buyLottos(lottoMachine, buyPrice);
```
지금까지 계속해서 이런 스타일을 구현하고 있음.
그런데 대부분의 다른 개발자들이 구현하는 스타일은 위와 같음.
앞으로 다른 개발자와의 소통을 위해 위와 같은 스타일로 구현하도록 습관을 바꿔보면 어떨까?
지금 바꾸지 않으면 추후에 변경하기 힘들 것으로 예상되어 피드백 남겨봄